appveyor.yml

```diff
@@ -34,4 +34,6 @@ build_script:
     Write-Output "Platform: $env:IdeVersion"
     & Setup\BuildInstallers.$env:IdeVersion.cmd
     if ($LastExitCode -ne 0) { $host.SetShouldExit($LastExitCode) }
+    #Get-ChildItem Setup\GitExtensions-*.msi | % { Push-AppveyorArtifact $_.FullName -FileName $_.Name }
```

Comment:
please remove if it is not needed

Reply:
I would like to provide an installer too. However, version should be increased too. 
I suggest adding a comment for now